docs/guide/pagination.md

```diff
@@ -159,6 +159,77 @@ query {
 > OffsetPaginated follow the same rules for the default pagination limit, and can be configured
 > in the same way as explained above.
 
+### Customizing queryset resolver
+
+It is possible to define a custom resolver for the queryset to either provide a custom
+queryset for it, or even to receive extra arguments alongside the pagination arguments.
+
+Suppose we want to pre-filter a queryset of fruits for only available ones,
+while also adding [ordering](./ordering.md) to it. This can be achieved like:
+
+```python title="types.py"
+
+@strawberry_django.type(models.Fruit)
+class Fruit:
+    name: auto
+    price: auto
+
+
+@strawberry_django.order(models.Fruit)
+class FruitOrder:
+    name: auto
+    price: auto
+
+
+@strawberry.type
+class Query:
+    @straberry.offset_paginated(OffsetPaginated[Fruit], order=order)
```

Comment:
Typo here @bellini666 but this is really gorgeous, overall, I'm loving this feature.